src/modules/index.ts

```diff
@@ -19,4 +19,4 @@ export * from "./xchain";
 export * from "./evm";
 export * from "./evmmerge";
 export * from "./feemarket";
-export * from "./vault";
+export * from "./perpsliquidity";
```

Comment:
As mentioned, not sure if it's a good idea to rename the vault stuff at this point.